package.json

```diff
@@ -10,13 +10,14 @@
     "build-samples": "yarn workspaces foreach --from '@monorepo/sample-*' run build",
     "run-samples": "yarn workspaces foreach -pvi --from '@monorepo/sample-*' run http-server",
     "lint": "yarn eslint packages",
-    "test": "jest --logHeapUsage -w 1",
+    "test": "jest --logHeapUsage",
     "test-print-config": "jest --showConfig",
     "test-e2e": "yarn workspace @monorepo/e2e-sample-app run test-e2e",
     "test-e2e-open": "yarn workspace @monorepo/e2e-sample-app run test-e2e-open",
     "test-e2e-ci": "start-server-and-test 'yarn run-samples' '9000|9001' 'yarn test-e2e'",
     "eslint": "eslint --max-warnings 0 --ext .js,.jsx,.ts,.tsx",
     "eslint-print-config": "eslint --print-config",
+    "jest": "TZ=UTC jest",
```

Comment:
I think it's generally better to set env. variables when calling scripts, rather than inside scripts or programs.

Side note, `patternfly-react` also sets the timezone this way, see [here](https://github.com/patternfly/patternfly-react/blob/main/package.json).